src/observer/models.py

```diff
@@ -190,3 +190,16 @@ def __str__(self):
 
     def __repr__(self):
         return '<ArticleJSON "%s">' % self
+
+class ProfileCount(models.Model):
+    total = PositiveIntegerField(null=False, blank=False)
+    timestamp = DateTimeField(auto_now_add=True)
```

Comment:
does this mean the time is generated on the Python object or in the database? The former usually gives better control and reproducibility, e.g. we are able to insert data retroactively if needed